usaspending_api/transactions/delta_models/summary_state_view.py

```diff
@@ -87,17 +88,37 @@
                 0
             ) AS NUMERIC(23, 2)
         ) AS face_value_loan_guarantee,
-        COUNT(*) AS counts
+        COUNT(*) AS counts,
+        CAST(
+            COALESCE(
+                SUM(matching_awards.total_outlays),
```

Comment:
This approach of summing the total outlays for matching awards will result in lot's of duplicate funds, and will not work. Many transactions are associated with the same awards, which means the funds associated with an award will be included in this sum for each associated transaction. 

We'll need to either dedupe the awards here or restructure the query to be based on Awards instead of Transactions.

Reply:
After a meeting discussion, we decided to move forward with the approach of running multiple queries where we will let the `summary_state_view` table get created and populated as usual, with the exception of creating the `total_outlays` column, but not calculating the values. Then in a subsequent query, we will use the `distinct_awards` from the `summary_state_view` table and map those to the `award_search` table to generate the total_outlays.